synapse/lib/stormsvc.py

```diff
@@ -92,15 +92,15 @@ class StormSvc:
     async def getStormSvcPkgs(self):
         return self._storm_svc_pkgs
 
-class StormSvcClient(s_base.Base, s_stormtypes.StormType):
+class StormSvcClient(s_base.Base, s_stormtypes.Proxy):
```

Comment:
?

Reply:
To allow a unified set of proxy methods to do toprim/fromprim rather than implementing yet another Method / GenrMethod detector